README.md

```diff
@@ -1,30 +1,19 @@
-# Optimizador de Restaurantes
+# Instalar
```

Comment:
¿Dónde dice que hay que explicar cómo instalar algo? ¿O make?

Reply:
¿Te refieres a nivel de issue o a nivel de los objetivos?

Me pareció útil comentar cómo instalar para que alguien que no entienda de Go o Make pueda empezar con el proyecto.

Reply:
Hazlo fuera de la evaluación del objetivo entonces. Os agradecería que pusiérais en esta sólo lo que sea evaluable y conduzca a la consecución del objetivo. Si no, sólo estáis añadiendo ruido.

Reply:
Entiendo entiendo, lo tengo en cuenta de ahora en adelante.

Gracias!